### DISCUSSION TITLE
FN3 Feedback

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#118] [Show and tell]
Author: @tdegeorge 
Reviewer: @sheppardw 

https://teambasedinquirylearning.github.io/precalculus/2024/FN3.html

**Comment by sheppardw** [comment]
I just typed a bunch of comments in a doc file. 
[FN Sec3 Feedback.docx](https://github.com/TeamBasedInquiryLearning/precalculus/files/15095154/FN.Sec3.Feedback.docx)
Let me know if you are not able to see these please. Still working on this.

**Reply by siwelwerd** [reply]
Copying it here so everyone can see it

> Objectives: Use the graph of a function to find the domain and range in interval notation, the x- and y-intercepts, the maxima and minimas, and where it is increasing and decreasing using interval notation.
>-Use math notation for x and y intercepts
>Activity 2.3.3a) There is only 1 x and y intercept, so do you want to say  What is the x intercept? Or “write the coordinates of the 
> x-intercept”.
> c) There is only 1 x and y intercept, so do you want to say  What is the y intercept? Or “write the coordinates of the y-intercept”.
> 
> Activity 2.3.6 a) Do you mean shade on a number line here?
> b)Did we introduce parentheses versus brackets yet? Maybe have a choice that is not including the end values.
> c) Shade on a number line? Possibly have a choice with a parenthesis.
> Figure 2.3.12: Can you make the hole a little larger. I don’t think it is totally clear that is an open circle.
> Figure 2.3.14: a)I don’t see the correct answer here for D because both endpoints look like closed points.
> b) I don’t see the correct answer here because the range looks like [-5,6].
> Activity 2.3.16. 
> Do you want both c) and d)? These look like the same question to me. Also, maybe put arrows on  the end of the curve?
> e) maxima or minima or just make it singular
> Defn 2.3.17: wording for “interval of increase” and decrease seems awkward.  Maybe use “ A function is increasing when the f(x) values increase as the x values also increase.” And “A function is decreasing when the f(x) values decrease as the x values increase.” You used “y-values” when describing a constant function, so I would just be consistent either way if you use f(x) or y values.
>

**Comment by sheppardw** [comment]
It looks really good overall!! The flow is good!

**Comment by tdegeorge** [comment]
Going to address each comment with a box of its own so I know where I am :)

When you say "use math notation" for the x and y-intercepts, what does that mean? I thought that if we had students write each of them as a coordinate pair, then that would be math notation?

**Reply by tdegeorge** [reply]
Oh - do you mean in the objective?

**Reply by sheppardw** [reply]
yes, just the code <m>

**Comment by tdegeorge** [comment]
For Activity 2.3.3 - I purposefully left "x-intercept(s)" knowing that one of the graphs only has one (and same with the y-intercept). I thought the phrasing would help students think more critically about whether the graphs have more than one x or y-intercepts.

**Reply by sheppardw** [reply]
That's fine!

**Comment by tdegeorge** [comment]
Activity 2.3.6: 
part a: I was trying to get students to think about all the potential x-values that would be a part of the domain. I didn't want to use the phrase "number line" because they are working on a coordinate plane....Would it be better to ask them to list all the values? I didn't want to say that initially because then it would exclude all the values of x in between....I added an instructor's note (under answer) here to help with making sure instructors understand the intent of this question. Also added in gridlines to help students see the domain and range values.

Parts b and d: Changed some answer choices so that they include parentheses as well.

**Reply by sheppardw** [reply]
I guess because it says "draw on the x axis". Are you wanting them to shade along the x axis in the same graph? I am just not sure what you want them to do. The reason being I usually end up having them make separate number lines for the x axis for more complicated domain questions, like combining functions, to get the visual (even when they may not have the entire graph). I hope that makes sense.

**Reply by tdegeorge** [reply]
Yes - I was thinking that they would shade on the graph that is given....

**Comment by tdegeorge** [comment]
Figure 2.3.12: I have no idea how to make the open circle in the graph larger. Every time I try, it doesn't make it clear that it is an open circle.... @StevenClontz - do you know how to do this?

**Reply by siwelwerd** [reply]
Change the 75 to whatever size you want.
```
p=arrow((3,-4), (-4,8),xmin=-6,xmax=5,ymin=-6,ymax=8, gridlines=[[-8..8],[-8..8]])
p+=point((3,-4), pointsize=75, markeredgecolor='blue',color='white',zorder=2)
p
```
![image](https://github.com/TeamBasedInquiryLearning/precalculus/assets/30658947/e0959735-cb3f-492a-9660-6b56ee82cf2e)

**Reply by tdegeorge** [reply]
It is still not showing up nice for me on this graph :(

**Comment by tdegeorge** [comment]
Figure 2.3.14: When I initially made this image, I wanted the right end point to be an open circle..... @StevenClontz - can you fix this one too?

**Reply by siwelwerd** [reply]
Instead of putting two points there, use one point with `markeredgecolor='blue',color='white'` like above.  You might need to increase the zorder to put it on top of the graph.

**Reply by siwelwerd** [reply]
```
p=line([(5,-1), (4,-5)],thickness=3,xmin=-6,xmax=8,ymin=-6,ymax=8, gridlines=[[-8..8],[-8..8]])
p+=line([(4,-5),(-1,6)],thickness=3)
p+=line([(-1,6),(-3,1)],thickness=3)
p+=point((-3,1),pointsize=50,color='blue')
p+=point((5,-1),pointsize=50,markeredgecolor='blue',color='white',zorder=2)
p
```
![image](https://github.com/TeamBasedInquiryLearning/precalculus/assets/30658947/67a130d4-3ebe-4db2-a65c-c6e41589b57a)

**Reply by tdegeorge** [reply]
I also attempted to fix this one but the preview is not showing it as nicely as yours....

**Comment by tdegeorge** [comment]
Activity 2.3.16: My intent in having parts a and c was to get students thinking about where the function is going "up" or "down" and then lead them into the mathematical notation with interval notation. Changed some of the wording...

**Reply by sheppardw** [reply]
Should there be arrows on the ends of the curve in the graph so that it is clear that it is negative infinity, etc.?

**Comment by tdegeorge** [comment]
Activity 2.3.19:
Part e: is there a way to put approximately in the answer choices? @siwelwerd @StevenClontz It is not quite at x=-2....

**Reply by sheppardw** [reply]
Maybe you could say "choose the x value closest to the"... max/min

**Comment by tdegeorge** [comment]
Definition 2.3.17: I changed the wording a little. See if this is better.

**Reply by sheppardw** [reply]
Looks good!

**Comment by tdegeorge** [comment]
Also put answers in for Activity 2.3.22. Some of these are not exact so I estimated....

**Comment by sheppardw** [comment]
2.3.18 minima and maxima (a for plural)

**Comment by sheppardw** [comment]
Activity 2.3.22
c) What "**are**" the x-intercept(s)
e) and f) do you want to specify using interval notation here or is inequality notation ok? Does it matter?
h) at what x value**(s)**

Looks very good!!
